### PR TITLE
added alignment stuff and some new functions. closes #14

### LINKED ISSUE #14: there should be a function to create the trialwise matrix from the sessionwise matrix. with parameters and stuff.
@m2kulkarni there should be a function to create the trialwise matrix from the sessionwise matrix. with parameters and stuff. It might not even need to be in the data, but generated by the user.
e.g. 
- I would like to create a matrix not from trial start, but from go cue, or reward, or the first lick every trial, paw movements etc.
- I want 10 seconds to be included from the previous trial

_Originally posted by @rozmar in https://github.com/kpdaie/BCI_analysis/issues/11#issuecomment-1154246684_

### PR DESCRIPTION
**Pull request by m2kulkarni**

### CODE REVIEW

**Comment by m2kulkarni** [comment]
@rozmar added the functions

**Comment by rozmar** [comment]
@m2kulkarni I made a few tweaks to your code, and added a pair of functions to do something like that in a more general way.
Your version respects trial starts and trial ends, the other solution doesn't care about other events, just the one you pick.

**Comment by rozmar** [comment]
Oh, I and I accidentally pushed the ROI alignment in this branch - it just does non-rigid transformation of the ROIs, so we can identify the conditioned neuron better.

**Comment by rozmar** [comment]
Ah, and the commit names are terrible..

File: BCI_analysis/io/io_suite2p.py [hunk 1]
```diff
@@ -1,11 +1,198 @@
...
+            start_frame = 0
+        else:
+            missing_frames_at_beginning = 0
+        F_aligned[missing_frames_at_beginning:missing_frames_at_beginning+end_frame-start_frame, :, i] = F[:, start_frame:end_frame].T
+    return F_aligned
+
+    
+
+
+def sessionwise_to_trialwise(F,
```
Comment (m2kulkarni) [inline BCI_analysis/io/io_suite2p.py:112]: does this function do something other than the top 2 combined? Or does this do the same thing as `align_trace_to_event`

**Comment by m2kulkarni** [comment]
looks good, merging

**Pull request merged**